lib/querytables.js

```diff
@@ -310,22 +310,18 @@ module.exports.getQueryMetadataModel = async function getQueryMetadataModel (pg,
     const statements = getQueryStatements(iSQL);
 
     const result = [];
-    try {
-        for (const query of statements) {
-            result.push(...await getAffectedTables(pg, query));
-        }
-    } catch (err) {
-        return callback(err);
+    for (const query of statements) {
+        result.push(...await getAffectedTables(pg, query));
```

Comment:
I'm gonna need you to monitor this in production to make sure requests are done one by one  and not all at the same time, which would create big spikes of connections.

Reply:
This portion of code remains the same before the refactor. You can check the behavior of `await` in a `for of` loop as follows:

```js
// file: for-async-test.js
'use strict'

function wait (ms) {
    return new Promise(resolve => setTimeout(resolve, ms));
}

async function main () {
    const times = [ 100, 100, 100 ];

    for (const [ index, ms ] of times.entries()) {
        console.log(`> loop ${index} before await`);
        await wait(ms);
        console.log(`> loop ${index} after await`);
    }
}

main();
```
should show:
```
$ node for-async-test.js
> loop 0 before await
> loop 0 after await
> loop 1 before await
> loop 1 after await
> loop 2 before await
> loop 2 after await
```